src/_includes/banner.html

```diff
@@ -1,32 +1,38 @@
-<section class="usa-banner" aria-label="Official government website">
+<section
+  class="usa-banner"
+  aria-label="Official website of the United States government"
+>
   <div class="usa-accordion">
     <header class="usa-banner__header">
       <div class="usa-banner__inner">
         <div class="grid-col-auto">
           <img
+            aria-hidden="true"
             class="usa-banner__header-flag"
             src="{{ '/assets/img/us_flag_small.png' | url }}"
             alt="U.S. flag"
```

Comment:
It looks like this alt tag should be empty. Our template leaves the alt tag for this and other icons empty.

```suggestion
            alt=""
```